scripts/docker/cross_compile/cross_compile.sh

```diff
@@ -25,8 +25,11 @@
 
 # Base docker image copies over the toolchain and rootfs, resulting from
 # the NASA_INSTALL.md instructions. Also installs minimum dependencies
-echo "Build context for base: "${ARMHF_CHROOT_DIR/../}
-docker build $ARMHF_CHROOT_DIR/../ -f scripts/docker/cross_compile/astrobee_base_cross.Dockerfile -t astrobee/astrobee:base-cross
+# echo "Build context for base: "${ARMHF_TOOLCHAIN}
```

Comment:
It looks like they are commented out?

Reply:
bah forgot to uncomment after doing some tests, fixed

Reply:
the code review is working :)